R/geo.R

```diff
@@ -58,7 +58,7 @@ setMethod("geo<-", c("CrunchVariable", "CrunchGeography"),
               geodata <- list(geodata = list(value))
 
               ent <- setEntitySlot(entity(x), "view", geodata)
-              dropCache(cubeURL(x))
+              dropCache(datasetReference(x))
```

Comment:
This seems overzealous. Adding geo data to a variable shouldn't require busting cache on everything related to the dataset, should it? What did you find that made dropping only cubes inadequate?